andela/krypton-ah-backend
# #163978352 add callback URL to API for account verification

#### What does this PR do?
It adds a callback URL to API for account verification process
#### Description of Task to be completed?
Add a callback URL to API for account verification purpose
#### How should this be manually tested?
* Enter localhost:3000/api/v1/users/signup on postman
* Select a POST method
* Provide the required inputs (firstname, lastname, email, password, callbackUrl(optional))
Click Send
* Check the verification mail sent to the provided email and click on the link
* After successful account verification, you will be redirected to the provided callback URL provided during signup
#### Any background context you want to provide?
Before now the application does not allow other applications to use the API for account verification which is not good
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/163978352

#### Screenshots (if appropriate)
<img width="1424" alt="screenshot 2019-02-14 at 4 50 04 pm" src="https://user-images.githubusercontent.com/20207295/52798636-9d9c8480-3078-11e9-9431-d5aa844f6aa4.png">

<img width="1421" alt="screenshot 2019-02-14 at 4 47 23 pm" src="https://user-images.githubusercontent.com/20207295/52798473-4a2a3680-3078-11e9-9009-78b1c1132605.png">

#### Questions:
